pandas/tests/io/parser/test_encoding.py

```diff
@@ -299,17 +299,21 @@ def test_readcsv_memmap_utf8(all_parsers):
     tm.assert_frame_equal(df, dfr)
 
 
-def test_not_readable(all_parsers, request):
+@pytest.mark.usefixtures("pyarrow_xfail")
+@pytest.mark.parametrize("mode", ["w+b", "w+t"])
+def test_not_readable(all_parsers, mode, request):
     # GH43439
     parser = all_parsers
-    if parser.engine in ("python", "pyarrow"):
+    if parser.engine == "pyarrow":
         mark = pytest.mark.xfail(
-            reason="SpooledTemporaryFile does only work with the c-engine"
+            reason="SpooledTemporaryFile does only work with the pyarrow-engine"
```

Comment:
umm is this reason correct? you mean does *not* right?

Reply:
thank you, fixed it :)